SpatialGDK/Source/Private/Interop/GlobalStateManager.cpp

```diff
@@ -188,3 +256,137 @@ bool UGlobalStateManager::IsSingletonEntity(Worker_EntityId EntityId)
 	}
 	return false;
 }
+
+void UGlobalStateManager::ToggleAcceptingPlayers(bool bInAcceptingPlayers)
```

Comment:
Can you give a bit of background on why a query is needed rather than just access the GSM with the initial entity ID?

Reply:
Also not a fan of `Toggle`. That implies switching the state from what it previously way. `Set` ?

Reply:
Changing the name to `SetAcceptingPlayers`.

When server travelling the old GSM will be destroyed so the initial entity ID will be changed so can't use that since we're toggling accepting players on the new GSM.

However we no longer need the entity query. AuthorityChanged will always keep the GSM entity ID up to date and so we don't need to find it. (was old code when this was being developed.)